api/tests/__init__.py

```diff
@@ -240,6 +249,7 @@ def setUpClass(cls):
         cls.centre_url = reverse("andela-centres-list")
         cls.country_url = reverse("countries-list")
         cls.department_url = reverse("departments-list")
+        cls.departmental_teams_url = reverse("departmental-teams-list")
```

Comment:
This is used only in the DepartmentTeamTestCase. We could move it there. Cleaning up this setup class can help improve the run time of tests significantly.